src/main/java/com/blamejared/ambientenvironment/AmbientEnvironment.java

```diff
@@ -16,96 +16,98 @@
 //            return blend(newColor, 0, (float) (value));
 //        };
     }
 
-    public static double remap(double value, double currentLow, double currentHigh, double newLow, double newHigh) {
+    public static double remap(final double value, final double currentLow, final double currentHigh, final double newLow, final double newHigh) {
         return newLow + (value - currentLow) * (newHigh - newLow) / (currentHigh - currentLow);
     }
 
-    private static float getRed(int hex) {
+    private static float getRed(final int hex) {
         return ((hex >> 16) & 0xFF) / 255f;
     }
 
-    private static float getGreen(int hex) {
+    private static float getGreen(final int hex) {
         return ((hex >> 8) & 0xFF) / 255f;
     }
 
-    private static float getBlue(int hex) {
+    private static float getBlue(final int hex) {
         return ((hex) & 0xFF) / 255f;
     }
 
-    private static float getAlpha(int hex) {
+    private static float getAlpha(final int hex) {
         return ((hex >> 24) & 0xff) / 255f;
     }
 
-    private static float[] getARGB(int hex) {
+    private static float[] getARGB(final int hex) {
         return new float[]{getAlpha(hex), getRed(hex), getGreen(hex), getBlue(hex)};
     }
 
-    private static int toInt(float[] argb) {
-        int r = (int) Math.floor(argb[1] * 255) & 0xFF;
-        int g = (int) Math.floor(argb[2] * 255) & 0xFF;
-        int b = (int) Math.floor(argb[3] * 255) & 0xFF;
-        int a = (int) Math.floor(argb[0] * 255) & 0xFF;
+    private static int toInt(final float[] argb) {
+        final int
+                r = (int) Math.floor(argb[1] * 255) & 0xFF,
+                g = (int) Math.floor(argb[2] * 255) & 0xFF,
+                b = (int) Math.floor(argb[3] * 255) & 0xFF,
+                a = (int) Math.floor(argb[0] * 255) & 0xFF;
         return (a << 24) + (r << 16) + (g << 8) + (b);
     }
 
-    public static double clamp(double value, double min, double max) {
+    public static double clamp(final double value, final double min, final double max) {
         return Math.max(min, Math.min(value, max));
     }
 
-    public static double curve(double start, double end, double amount, double waves) {
+    public static double curve(final double start, final double end, double amount) {
         amount = clamp(amount, 0, 1);
         amount = clamp((amount - start) / (end - start), 0, 1);
         return clamp(0.5 + 0.5 * Math.sin(Math.cos(Math.PI * Math.tan(90 * amount))) * Math.cos(Math.sin(Math.tan(amount))), 0, 1);
     }
 
-    public static int blend(int color1, int color2, float ratio) {
-        float ir = 1.0f - ratio;
+    public static int blend(final int color1, final int color2, final float ratio) {
+        final float ir = 1.0f - ratio;
 
-        float[] rgb1 = getARGB(color2);
-        float[] rgb2 = getARGB(color1);
+        final float[] rgb1 = getARGB(color2), rgb2 = getARGB(color1);
```

Comment:
Please can you change this back to how it was before, making it a single declaration like this doesn't help anything and just makes the code harder to read.